tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java

```diff
@@ -927,7 +884,7 @@ public String getOrgCoverageLevel(String locale) {
             level = WebContext.COVLEV_DEFAULT_RECOMMENDED_STRING;
         } else {
             org.unicode.cldr.util.Level l = StandardCodes.make().getLocaleCoverageLevel(myOrg, locale);
```

Comment:
I know it's not part of this review, but damn that looks like a '1' in my font. The snippet "(l == org.unicode.cldr.util.Level.UNDETERMINED)" is particularly easy to misread.
Perhaps change it to "coverageLevel" ?

Reply:
glad to ob1ige